### PR TITLE
feat(node): Support `Buffer.readUIntLE`

### PR DESCRIPTION
**Pull request by getspooky**
Reads byteLength number of bytes from buf at the specified offset and interprets the result as an unsigned, little-endian integer supporting up to 48 bits of accuracy.

### CODE REVIEW

**Comment by talentlessguy** [comment]
related: #1326

File: node/buffer.ts [hunk 1]
```diff
@@ -273,6 +273,20 @@ export class Buffer extends Uint8Array {
     this.boundsError(byteLength, 4, "byteLength");
   }
 
+  /**
+   * Reads byteLength number of bytes from buf at the specified offset and interprets
+   * the result as an unsigned, little-endian integer supporting up to 32 bits of accuracy.
```
Comment (kt3k) [inline node/buffer.ts:278]: I think doc should say `supporting up to 48 bits of accuracy` ref: https://nodejs.org/api/buffer.html#buffer_buf_readuintle_offset_bytelength

The purpose of std/node is the compatibility to node.js, not fixing it or providing our own design.

Comment (getspooky) [reply node/buffer.ts]: Yes, i will fixe that.

File: node/buffer_test.ts [hunk 2]
```diff
@@ -299,7 +299,7 @@ Deno.test({
 });
 
 Deno.test({
-  name: "Buffer readUIntBE",
```
[the remark this answers is not in the document]
Comment (getspooky) [reply node/buffer_test.ts]: Yes, i will add a separate test for `readUIntLE`.

**Review by kt3k — APPROVED** [review]
Thank you for updating! @getspooky 

LGTM

**Pull request merged**